package.json

```diff
@@ -30,10 +30,15 @@
   "author": "Idan Tovi",
   "license": "MIT",
   "dependencies": {
-    "ajv": "^5.5.2",
-    "swagger-parser": "^4.0.1"
+    "ajv": "^6.6.2",
+    "koa-multer": "^1.0.2",
+    "lodash": "^4.17.11",
```

Comment:
you only use two functions from entire lodash. Are you sure there are no more lightweight alternatives?
(in general I'm quite pro-Lodash, but in libraries it's better to avoid bringing additional heavy transitive dependencies when possible)